src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
+import enums.Colors;
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random;
+
+    public ColorSupplier() {
+        this.random = new Random();
+    }
+
     public String getRandomColor() {
-        return null;
+        int colorIndex = random.nextInt(Colors.values().length);
+        return Colors.values()[colorIndex].toString();
+    }
+
+    public Random getRandom() {
```

Comment:
delete this method please

Reply:
++

Reply:
i couldn't do it because when ran the command 'mvn clean package' it thrown the error that field 'random' has to be private and have the accessor method. that is why this getter is here.
what can i do here?